admin/build_dev_wheels.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+REPO_ROOT="$(realpath "$(dirname $0)/..")"
+
+# if you edit this or specify a new value make sure the packages are in dependency order!
+REPOS=${REPOS-archinfo pyvex claripy cle ailment angr}
+
+OUTDIR=${OUTDIR-./build}
+mkdir -p "$OUTDIR"
+
+python -m pip install -U setuptools wheel build
+export PIP_FIND_LINKS=$OUTDIR
+
+for REPO in $REPOS; do
+	python -m build --wheel --outdir "$OUTDIR" "$REPO_ROOT/$REPO"
```

Comment:
If you remove `--wheel`, it will build both sdists and wheels, which might be helpful in some multi-platform circumstances, but will be larger on disk. Up to you.

Reply:
That's a good idea.